Apollo/Windows/PreferencesWindow.xaml

```diff
@@ -126,6 +126,30 @@
               <Components:VerticalAdd AlwaysShowing="true" ToolTip.Tip="Create Virtual Launchpad" Added="Launchpad_Add" />
             </StackPanel>
             
+            <Border />
+            
+            <TextBlock Classes="heading" Text="User Interface Motion" />
+            <Grid ColumnDefinitions="Auto,*">
+              <StackPanel Grid.Column="0" Orientation="Horizontal" Spacing="4">
+                <TextBlock VerticalAlignment="Center" Text="Motion:" />
+                <ComboBox VerticalAlignment="Center" x:Name="UIMotion" SelectionChanged="UIMotion_Changed">
+                  <ComboBoxItem>Off</ComboBoxItem>
+                  <ComboBoxItem>Reduced</ComboBoxItem>
+                  <ComboBoxItem>Full</ComboBoxItem>
+                </ComboBox>
+              </StackPanel>
+              <TextBlock Grid.Column="1" Classes="heading" Margin="5 0 0 0" HorizontalAlignment="Center" VerticalAlignment="Center" x:Name="UIMotionWarning" />
+            </Grid>
+            <StackPanel Orientation="Horizontal" Spacing="4">
+              <TextBlock VerticalAlignment="Center" Text="Hover Time:" />
+              <ComboBox VerticalAlignment="Center" x:Name="UIHoverTime" SelectionChanged="UIHoverTime_Changed">
+                <ComboBoxItem>None</ComboBoxItem>
+                <ComboBoxItem>Short</ComboBoxItem>
+                <ComboBoxItem>Medium</ComboBoxItem>
+                <ComboBoxItem>Long</ComboBoxItem>
```

Comment:
I would prefer calling this "Animation Speed" with options "Instant", "Fast", "Normal"(?), "Slow"